Composer/packages/client/src/pages/home/ItemContainer.tsx

```diff
@@ -16,8 +18,12 @@ import {
   childrenContainer,
 } from './styles';
 
-interface ItemContainerProps extends Omit<IButtonProps, 'onChange' | 'styles' | 'title'> {
-  onClick?: () => void | Promise<void>;
+const { openExternal } = window;
+
+type onClickFunctionType = () => void | Promise<void>;
+
+interface ItemContainerProps {
+  onClick?: onClickFunctionType | boolean;
```

Comment:
Instead of abusing booleans for this, what if we defined a single-valued type in here and then imported that type/value in `index.tsx` for use in the onClick prop? Or used a Symbol?

Reply:
we cleaned this up.